.github/workflows/phpCS.yml

```diff
@@ -0,0 +1,21 @@
+name: PHP Code Style
+
+on: [push]
+
+jobs:
+    phpcs:
+        name: PHP CodeSniffer
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v1
+
+            - name: Setup PHP
+              uses: nanasess/setup-php@49cb618e1f511dc9ca5c3a56ad4aabdbf69a61fe
```

Comment:
I am curious if `nanasess/setup-php` is chosen because of wider PHP legacy verison support (5.4/5.5)... I don't think this is necessary any more.

`shivammathur/setup-php` looks more popular (and has more contributors)? (it has most stars [in the market](https://github.com/marketplace?after=Y3Vyc29yOjIw&query=php&type=actions&utf8=%E2%9C%93))

Reply:
Mh, for some reason I was not aware that o.O
Fixing it now...